pkg/csi/service/common/placementengine/placement.go

```diff
@@ -99,6 +102,7 @@ func GetSharedDatastores(ctx context.Context, reqParams interface{}) (
 				// If there are preferred datastores among the compatible
 				// datastores, choose the preferred datastores, otherwise
 				// choose the compatible datastores.
+				log.Infof("filtering preferential datastores from compatible datastores")
```

Comment:
`filtering` ==> `Filtering`

Reply:
Can be debug log. Since there is no return paths between this log and previous log. 
Both the logs will be printed all the time. Would prefer to keep one of them

Reply:
marked as Debug log. Just wanted to have some log lines to quickly identify if there is any preferred data store we have to filter.